ui/css/perf.css

```diff
@@ -106,10 +106,13 @@ to { opacity: 0; }
     background-image: url("../img/tip-locked.png");
 }
 
-#close-popup {
-  float:right;
-  display:inline-block;
-  cursor:pointer;
+.graphchooser-close {
+  color: #fff;
+  outline: none;
```

Comment:
Could you add a `margin: -14px -4px;` here. I think it looks a little better up and to the right.